src/Entity/ApiConfiguration.php

```diff
@@ -120,14 +118,14 @@ public function setApiClientSecret(string $apiClientSecret): self
         return $this;
     }
 
-    public function isEnterprise(): ?bool
```

Comment:
IMHO, it would be BC Break to remove it now ? 

Let keep it for now, and do the check on getEdition ?